Hotfix/v1.1.0: Updated the Changelogfile (#52)
XgridInc/xc3: CHANGELOG.md

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.1.0]  - 2023-07-17

### Changed
- Integrated EIC endpoint and removed bastion host
- Updated the infra-deployment based on environment variable
- Added the automation script to deploy XC3 infrastack
- Updated the Architecture Diagram of XC3 in the Readme file

### Fixed
- Fixed Linked List Lambda to run from a member account as well
- Added checks for the packages installation [AWS CLI, python, terraform] in the init.sh bash script.

> 

Reviewer: ```suggestion
```

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.1.0]  - 2023-07-17

### Changed

> - Added the EIC endpoint

Reviewer: ```suggestion
- Integrated EIC endpoint and removed bastion host
```

> - Added env condition for infra

Reviewer: ```suggestion
- Updated the infra-deployment based on environment variable
```

> - Added the automation script

Reviewer: ```suggestion
- Added the automation script to deploy XC3 infrastack
```

> - Updated the Architecture Diagram in the Readme file

Reviewer: ```suggestion
- Updated the Architecture Diagram of XC3 in the Readme file
```

> - Updated the Readme file

Reviewer: ```suggestion
```

### Fixed

> - Linked List Lambda issue fixed

Reviewer: ```suggestion
- Fixed Linked List Lambda to run from a member account as well
```

> - Added checks for the packages [AWS cli, pyhton, terraform] in the init.sh bash script.

Reviewer: ```suggestion
- Added checks for the packages installation [AWS CLI, python, terraform] in the init.sh bash script.
```

and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [1.1.0]  - 2023-07-17

### Changed
- Added the EIC endpoint
- Added env condition for infra
- Added the automation script
- Updated the Architecture Diagram in the Readme file
- Updated the Readme file

### Fixed
- Linked List Lambda issue fixed

> - Fixed some issues in the pre-req.sh bash script

Reviewer: Mention the issues as well

Author: Added in 51cb5a12dd43a8abb2932db2fa4de5d270cb1edf
